### PR TITLE
GH 172 | Adding import to readme

### PR DESCRIPTION
**Pull request by charliesantos**
https://github.com/twilio/twilio-client.js/issues/172

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,14 @@ We recommend using `npm` to add the Client SDK as a dependency.
...
 
-Using this method, you can `require` twilio-client.js like so:
+Using this method, you can `import` twilio-client.js like so:
+
+```js
+import { Device } from 'twilio-client';
+
+```
+
+Or using `require`:
```
Comment (ryan-rowland) [inline README.md:35]: When would a customer use `import` and when would they use `require`? We should clarify why there are two ways to do this.

Comment (charliesantos) [reply README.md:35]: What do you think about removing the section about `require`? I think if we leave just the `import` section, it's pretty much straight forward.

Comment (ryan-rowland) [reply README.md:35]: We can probably just say something like "In ES6 and newer, and in TypeScript..." and "In older JS syntax..." and it should be clear enough

Comment (charliesantos) [reply README.md]: I updated the readme, please check again.

**Pull request merged**